Makefile

```diff
@@ -16,23 +16,17 @@ GIT_SHA=$(shell git rev-parse HEAD)
 GOFLAGS=-ldflags "-X github.com/edgexfoundry/device-mqtt-go.Version=$(VERSION)"
 
 build: $(MICROSERVICES)
-	go build ./...
+	$(GO) build ./...
 
 cmd/device-mqtt:
 	$(GO) build $(GOFLAGS) -o $@ ./cmd
 
 test:
-	go test ./... -cover
+	$(GO) test ./... -cover
 
 clean:
 	rm -f $(MICROSERVICES)
 
-prepare:
```

Comment:
So the Jenkins job has been modified to not call this `make` step then?

Reply:
yes